docs/conf.py

```diff
@@ -28,9 +28,11 @@
 pygments_style = 'sphinx'
 
 intersphinx_mapping = {
-    'http://docs.python.org/2.7': None,
-    'http://docs.djangoproject.com/en/1.9':
+    'python': ('http://docs.python.org/3', None),
```

Comment:
```suggestion
    'python': ('http://docs.python.org/', None),
```

Maybe?

Reply:
I changed it as you suggest, but note that it results in this message while building the docs:

```
intersphinx inventory has moved: http://docs.python.org/objects.inv -> https://docs.python.org/3/objects.inv
```

... so let me know if you want me to change it back in light of that.

Reply:
In that case please revert it, sure. Didn't know.

But updating the protocol to https in all URLs would make sense, wouldn't it?

Reply:
OK, done on both counts.